src/tools/tidy/src/issues.txt

```diff
@@ -390,26 +391,29 @@
 "ui/borrowck/issue-95079-missing-move-in-nested-closure.rs",
 "ui/box/issue-82446.rs",
 "ui/box/issue-95036.rs",
+"ui/c-variadic/issue-32201.rs",
+"ui/c-variadic/issue-86053-1.rs",
+"ui/c-variadic/issue-86053-2.rs",
```

Comment:
The new order was not quite stable for this round, but it should be in the future, as BTreeSet yields items in order when iterated and uses the Ord of String.